README.md

```diff
@@ -10,10 +10,34 @@ It's totally not the first approach of this kind. But: this one should be easily
 
 Currently, Wireshell  consists of the following basic commands:
 
-#### New
+#### Fresh installation
```

Comment:
Thank you

*Hari K T*

You can ring me : +91 9388 75 8821

http://harikt.com , https://github.com/harikt ,
http://www.linkedin.com/in/harikt , http://www.xing.com/profile/Hari_KT

Skype  : kthari85
Twitter : harikt

On Mon, Apr 6, 2015 at 8:32 PM, Marcus Herrmann <notifications@github.com>
wrote:

> In README.md
> <https://github.com/marcus-herrmann/wireshell/pull/2#discussion_r27802240>
> :
>
> > @@ -10,10 +10,34 @@ It's totally not the first approach of this kind. But: this one should be easily
> >
> >  Currently, Wireshell  consists of the following basic commands:
> >
> > -#### New
> > +#### Fresh installation
>
> That was not exactly what I meant, but anyway, it's just a detail and I'll
> merge (and maybe modify later)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/marcus-herrmann/wireshell/pull/2/files#r27802240>.
>